build_tools/services.rb

```diff
@@ -10,10 +10,10 @@ class ServiceEnumerator
     MANIFEST_PATH = File.expand_path('../../services.json', __FILE__)
 
     # Minimum `aws-sdk-core` version for new gem builds
-    MINIMUM_CORE_VERSION = "3.188.0"
+    MINIMUM_CORE_VERSION = "3.190.1"
```

Comment:
Do we need to bump this since smoke tests/features are not part of the built gems?

Reply:
No, but they will bump anyway....